uportal-portlets-overlay/cas-proxy-test-portlet/pom.xml

```diff
@@ -42,6 +42,20 @@
             <version>${cas-proxy-test-portlet.version}</version>
             <type>war</type>
         </dependency>
+        <dependency>
+            <groupId>org.jasig.portlet</groupId>
+            <artifactId>cas-proxy-test-portlet</artifactId>
+            <version>${cas-proxy-test-portlet.version}</version>
+            <type>pom</type>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
yes it is, it's needed to build the classe provided for overriding (until my PR on it is integrated and test-proxy-cas-portlet version was upgraded here)